docs/conf/portland/2024/writing-day.rst

```diff
@@ -189,6 +189,22 @@ We're particularly interested in:
 
 This is where we need you the most! Everyone who works in documentation is a subject matter expert on this topic, and we want your ideas on how to make the survey better benefit the community as a whole.
 
+Tabletop game rules writing workshop
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Project Organizer Ryan Macklin, they/he. Afternoon only session.
+
+Tabletop game rules are a neat and weird form of technical documentation! 
+
+If you're working on writing tabletop game rules or are interested in learning about that, join me during the Writing Day afternoon session. You can bring your own work-in-progress rules documents, and I'll have some work-in-progress rules as well. 
+
+The purpose for this Writing Day workship is to help:
+
+* Technical writers, who are also tabletop game writers, with their rules writing
```

Comment:
Please revise to: Technical writers turning their craft toward game rules writing

(The commas here suggest a different intent. This should hopefully be clearer)